projects/allinone/src/main/java/org/batfish/allinone/AllInOne.java

```diff
@@ -167,8 +168,21 @@ public void run() {
   }
 
   private void runCoordinator() {
-
-    final String[] argArray = getArgArrayFromString(_settings.getCoordinatorArgs());
+    String coordinatorArgs =
+        String.format("%s", _settings.getCoordinatorArgs());
```

Comment:
is the `String.format` call here not a no-op?

Reply:
Yes, it was. Good catch.